csharp-intermediate-topics/TupleAliasInCSharp/TupleAliasInCSharp/GlobalAlias/GlobalAlias.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace TupleAliasInCSharp.GlobalAlias 
```

Comment:
Let's use file-scoped namespaces.

Reply:
This PR has been updated to reflect this change.